arithmetization/src/main/java/net/consensys/linea/config/LineaRpcConfiguration.java

```diff
@@ -18,8 +18,17 @@
 import java.math.BigDecimal;
 
 import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.Accessors;
 
 /** The Linea RPC configuration. */
 @Builder(toBuilder = true)
-public record LineaRpcConfiguration(
-    boolean estimateGasCompatibilityModeEnabled, BigDecimal estimateGasCompatibilityMultiplier) {}
+@Accessors(fluent = true)
+@Getter
+@ToString
+public class LineaRpcConfiguration {
+  @Setter private volatile boolean estimateGasCompatibilityModeEnabled;
+  private BigDecimal estimateGasCompatibilityMultiplier;
```

Comment:
The way it is, it's probably Ok, but with the next [feature](https://github.com/Consensys/zkevm-monorepo/issues/3273) it might need to be volatile. But I might be wrong, I'm not as familiar with this codebase